external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -131,6 +150,21 @@ private void failQueuedTuples(List<Tuple> failedTuples) {
         return queuedTuples;
     }
 
+    @Override
+    public Map<String, Object> getComponentConfiguration() {
```

Comment:
@vesense  I noticed that the body of this method is exactly the same for AbstractHdfsBolt. Furthermore, it's very likely that many Storm connector bolts will use the same code. Can you please write a helper method in TupleUtils and call that method here and AbstractHdfsBolt.